spec/models/user_spec.rb

```diff
@@ -1,4 +1,31 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
+  let(:user) do
+    User.new(email: "joe@example.com", user_name: "joe", password: "pass")
+end
+
+  it 'is valid' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.